slides/intro-03-what-makes-a-model.qmd

```diff
@@ -387,26 +394,24 @@ plot_rates +
 
 ::: columns
 ::: {.column width="50%"}
-### Logistic regression
-```{r plot-logistic-reg-3}
+### PH model
+```{r plot-ph-3}
 #| echo: false
 #| fig.width: 7
 #| fig.height: 6
 
-plot_rates +
-  geom_line(aes(A, .pred_one, col = I(test_color)), linewidth = 2, alpha = 0.8, data = logistic_preds)
+p_ph
 ```
 :::
 
 ::: {.column width="50%"}
-### Decision trees
+### Decision tree
```

Comment:
Do you deliberatively make the titles different ("tree" vs "trees" on the third slide in the sequence)?

Reply:
Nope, that was a left over from the previous version, I decided to switch to the singular form here. Thanks for the catch!